breaking-front/src/components/Modal/Modal.js

```diff
@@ -10,6 +10,21 @@ export default function Modal({ isOpen, closeClick, title, grid, children }) {
   const modalOverlayClick = (event) => {
     if (event.target === modalRef.current) closeClick();
   };
+  const scrollY = window.scrollY;
+  useEffect(() => {
+    if (isOpen) {
+      document.body.style.cssText = `
+      position: fixed; 
+      top: -${scrollY}px;
+      width: 100%;
+      `;
+    }
+    return () => {
+      const scrollY = document.body.style.top;
+      document.body.style.cssText = '';
+      window.scrollTo(0, parseInt(scrollY || '0', 10) * -1);
+    };
```

Comment:
return 부분 로직은 첫 렌더링 시와 modal이 켜질 때와 꺼질 때 둘 다 실행하지만, 
밑의 방식으로 하면 첫 렌더링 시와 Modal이 꺼질 때에만 실행됩니다.
```suggestion
  useEffect(() => {
    if (isOpen) {
      document.body.style.cssText = `
      position: fixed; 
      top: -${scrollY}px;
      width: 100%;
      `;
    } else {
      const scrollY = document.body.style.top;
      document.body.style.cssText = '';
      window.scrollTo(0, parseInt(scrollY || '0', 10) * -1);
    }
```

그리고 scrollY가 13번째 줄에서도 동일한 이름으로 선언되어있어 혼동이 있을거 같습니다. 두 변수의 이름을 다르게 수정 부탁드립니다😀